src/modules/fare-contracts/details/Barcode.tsx

```diff
@@ -249,28 +250,31 @@ const StaticQrCode = ({fc}: {fc: FareContractType}) => {
   );
 };
 
-const useStyles = StyleSheet.createThemeHook((theme) => ({
-  aztecCode: {
-    width: '100%',
-    aspectRatio: 1,
-    padding: theme.spacing.large,
-    backgroundColor: '#FFFFFF',
-    maxHeight: 275,
-  },
-  staticBottomContainer: {
-    flex: 1,
-    backgroundColor: '#ffffff',
-  },
-  staticQrCode: {
-    padding: 0,
-    marginLeft: 'auto',
-    marginRight: 'auto',
-    maxWidth: 250,
-  },
-  staticQrCodeSmall: {
-    maxWidth: 125,
-  },
-}));
+const useStyles = () => {
+  const {aztec_code_max_height, aztec_code_padding} = useRemoteConfigContext();
```

Comment:
Wonder if this warrants a comment actually. 

> "why in all that is holy is this configurable" 
– someone in 2 years

I don't think the need here is self explanatory, so describing why this is needed I think is nice. "Some barcode readers are sensitive on size, etc etc blah blah"

Reply:
I was just about to ask why (in all that is holy) the padding is configurable. Is the height not enough?

Reply:
Might want to add a note in the comment on darkmode/lightmode. There is some concern that the surrounding white border field in darkmode has implications to how well the barcode is read. Meaning, if it is too small it is harder to read. This isn't really confirmed (in my mind), but still being able to experiment to be a bit more empirical is nice.

Note: Testing all this locally just wont do and we've used several hours trying to check read stability/speed, but out in the real world a lot of things happen (different devices, different readers, sun, movement etc etc)

Reply:
Could be clever with maths etc, but I think it is just as simple and versatile to just treat the values separately